app/src/main/java/org/dhis2/usescases/main/program/ProgramFragment.kt

```diff
@@ -78,11 +83,32 @@ class ProgramFragment : FragmentGlobalAbstract(), ProgramView {
     override fun onResume() {
         super.onResume()
         presenter.init()
+        val gd: GradientDrawable = binding.drawerLayout?.background as GradientDrawable
+        ValueAnimator.ofInt(0, 16.dp)
+            .apply {
+                duration = 700
+                startDelay = 700
+                interpolator = OvershootInterpolator()
+                addUpdateListener {
+                    val value = (it.animatedValue as Int).toFloat()
+                    gd.cornerRadii = floatArrayOf(value, value, value, value, 0f, 0f, 0f, 0f)
+                }
+            }.start()
```

Comment:
is this animation new? should it be here????

Reply:
true, it would be better in its own class